docs/ops.html

```diff
@@ -1411,18 +1411,18 @@ <h4 class="anchor-heading"><a id="remote_jmx" class="anchor-link"></a><a href="#
       </tr>
       <tr>
         <td>Message in rate</td>
-        <td>kafka.server:type=BrokerTopicMetrics,name=MessagesInPerSec</td>
-        <td></td>
+        <td>kafka.server:type=BrokerTopicMetrics,name=MessagesInPerSec,topic=([-.\w]+)</td>
+        <td>Number of incoming messages per topic.</td>
```

Comment:
The third column's header is `Normal value` so it feels a bit weird to put a description in there. The description is actually the first column. Perhaps, we could complement the description when necessary instead of adding this?